.git-config/hooks/pre-commit

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+LOCAL_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+
+RENOVATE_PREFIX="^renovate/"
+
+VALID_PREFIXES="^(feat|spike|fix|test|refactor)\/(.*)"
+VALID_TICKET_1='CIV-[0-9]{1,5}$'
+VALID_TICKET_2='CIV-[0-9]{1,5}[-_][a-zA-Z0-9]+.*'
```

Comment:
I had a little change of heart on this one, as in the past I had people submitting two different branches with identical names safe for an uppercase letter and this caused me some headaches.

Can I ask that this expression be modifed as follows?

```suggestion
VALID_TICKET_2='CIV-[0-9]{1,5}[-_][a-z0-9]+.*'
```

Reply:
Done